python/graphstorm/dataloading/dataloading.py

```diff
@@ -199,154 +205,167 @@ def __init__(self, dataset, target_idx, fanout,
-        Batch size
+        Mini-batch size.
     label_field: str or dict of str
         Label field of the edge task.
-    node_feats: str, or dist of list of str
-        Node features.
-        str: All the nodes have the same feature name.
-        list of string: All the nodes have the same list of features.
-        dist of list of string: Each node type have different set of node features.
-        Default: None
-    edge_feats: str, or dist of list of str
-        Edge features.
-        str: All the edges have the same feature name.
-        list of string: All the edges have the same list of features.
-        dist of list of string: Each edge type have different set of edge features.
-        Default: None
-    decoder_edge_feats: str or dict of list of str
-        Edge features used in decoder.
-        str: All the edges have the same feature name.
-        list of string: All the edges have the same list of features.
-        dist of list of string: Each edge type have different set of edge features.
-        Default: None
+    node_feats: str, or dict of list of str
+        Node feature fileds in three possible formats:
+
+            - string: All nodes have the same feature name.
+            - list of string: All nodes have the same list of features.
+            - dict of list of string: Each node type have different set of node features.
+
+        Default: None.
+    edge_feats: str, or dict of list of str
+        Edge features fileds in three possible formats:
+
+            - string: All edges have the same feature name.
+            - list of string: All edges have the same list of features.
+            - dict of list of string: Each edge type have different set of edge features.
+
+        Default: None.
+    decoder_edge_feats: str, or dict of list of str
+        Edge features used in decoder in three possible formats:
+
+            - string: All edges have the same feature name.
+            - list of string: All edges have the same list of features.
+            - dict of list of string: Each edge type have different set of edge features.
+
+        Default: None.
     train_task : bool
-        Whether or not for training.
+        Whether or not is the dataloader for training.
     reverse_edge_types_map: dict
-        A map for reverse edge type
+        A map for reverse edge type.
     exclude_training_targets: bool
-        Whether to exclude training edges during neighbor sampling
+        Whether to exclude training edges during neighbor sampling.
     remove_target_edge_type: bool
-        Whether we will exclude all edges of the target edge type in message passing.
+        Whether to exclude all edges of the target edge type in message passing.
     construct_feat_ntype : list of str
         The node types that requires to construct node features.
     construct_feat_fanout : int
-        The fanout required to construct node features.
+        The fanout that requires to construct node features.
```

Comment:
Who requires what?
How about: The fanout used when constructing node features for feature-less nodes.

Reply:
Updated to `The fanout used when constructing node features for feature-less nodes.`